docs/install-and-run.md

```diff
@@ -121,86 +131,88 @@ In cases where CocoaPods is not compatible with your project, you can add EarlGr
 
 #### Step 1: Generate EarlGrey.framework
 
-  1. Download the source for EarlGrey and its dependencies.
-     * Download EarlGrey's source code from the [Latest Release](https://github.com/google/EarlGrey/archive/1.0.0.zip)
+1. Download the source for EarlGrey and its dependencies.
+  * Download EarlGrey's source code from the [Latest Release](https://github.com/google/EarlGrey/archive/1.0.0.zip)
```

Comment:
1.0 is not the latest release. Use this link instead:

`https://github.com/google/EarlGrey/releases/latest`